server/plugin.go

```diff
@@ -426,6 +428,38 @@ func (p *Plugin) OnActivate() error {
 		}
 	}
 
+	remoteID, err := p.API.RegisterPluginForSharedChannels(model.RegisterPluginOpts{
+		Displayname:  "ms_teams_connect",
+		PluginID:     pluginID,
+		CreatorID:    botID,
+		AutoShareDMs: true,
+		AutoInvited:  true,
```

Comment:
Can you remind me what this does?

Reply:
This doesn't require to go through the Invite process that shared channels require.